sixtyfps_runtime/rendering_backends/qt/widgets.rs

```diff
@@ -177,15 +178,23 @@ impl Item for NativeButton {
         _window: &WindowRc,
     ) -> LayoutInfo {
         let mut text: qttypes::QString = self.text().as_str().into();
+        let icon = crate::qt_window::load_image_from_resource(
```

Comment:
```suggestion
        let icon : qttypes::QPixmap = crate::qt_window::load_image_from_resource(
```

I like to have the type specified near the cpp! macro because the cpp macro does only minimal type checking (same size, basically) so settinh the type explicitly make sure that one can quickly see that this is correct.